templates/layouts/_questionnaire.html

```diff
@@ -37,4 +37,14 @@
           })
       }}
   {% endblock %}
+
+  {% block cant_complete %}
```

Comment:
This `cant_complete` block can be removed as it's not overridden anywhere.

Reply:
Thanks, removed.